alica_ros_timer/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
don't need a separate package for this. Could be part of alica_ros_proxy which already has clock and communication.